src/commands/remove.js

```diff
@@ -178,6 +191,13 @@ export default async function main(ctx) {
     `${chalk.bold(contextName)} ${elapsed(Date.now() - findStart)}`
   );
 
+  if (deployments.length > 200) {
+    output.warn(
+      `Only 200 deployments can get deleted at once. ` +
+      `Please continue 10 minutes after deletion to remove the rest.`
```

Comment:
@AndyBitz you can get the exact reset time from here: https://zeit.co/docs/api#api-basics/rate-limits